test/benchmarks/quantum_info.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=missing-docstring,invalid-name,no-member
+# pylint: disable=attribute-defined-outside-init
+
+from qiskit.quantum_info import random_clifford, Clifford, decompose_clifford
+from qiskit.quantum_info import random_cnotdihedral, CNOTDihedral
+import numpy as np
+
+
+class RandomCliffordBench:
+    params = ['1,3000', '2,2500', '3,2000', '4,1500', '5,1000', '6,700']
+    param_names = ['nqubits,length']
```

Comment:
So I missed this before. I think doing it as a string is fine, but the normal asv pattern for this would be something like:

```suggestion
    params = [[1, 2, 3, 4, 5, 6], [3000, 2500, 2000, 1500, 1000, 700]
    param_names = ['nqubits', 'length']
```

and then in the setup method raise a `NotImplementedError` for the combinations you don't want to run. But that gets messy since there is only a single pairing so this is nicer.